src/index.css

```diff
@@ -97,3 +97,7 @@ input:focus-visible {
 .dangerouslySetInnerHTML li {
   margin-left: 23px;
 }
+
+h1:focus-visible {
+  outline: 2px solid transparent;
+}
```

Comment:
pffff c'est un peu nul 
Le but c'est de supprimer le cadre qui 'apparait au chargement d'une page sur le titre. 
ça vient du fait que AccessibleNavigation.tsx force le focus sur le h1 pour une fin d'accessibilité 
https://a11y-guidelines.orange.com/en/articles/single-page-app/#notify-user-of-page-changes

C'est un comportment qui me parait pas ouf mais je sais pas quelle est la bonne pratique. 
Bref du coup ici je surcharge juste le style de cette mise en focus.  Si on a le temps on creusera cette question pour l'accesibilité

Reply:
Oui effectivement ça fait un peu arnaque ton affaire mais bon ça sera plus joli comme ça